analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/CollectorTestCheck.cs

```diff
@@ -33,6 +33,8 @@ internal class CollectorTestCheck : SymbolicCheck
         public readonly List<SymbolicContext> PostProcessed = new();
         private readonly List<(string Name, SymbolicContext Context)> tags = new();
 
+        private int exitReachedCount;
```

Comment:
_Nitpick:_ the preceding empty line doesn't need to be there. It's a series of fields.